pkg/epforwarder/epforwarder.go

```diff
@@ -39,6 +40,7 @@ var passthroughPipelineDescs = []passthroughPipelineDesc{
 		eventType:              eventTypeDBMMetrics,
 		endpointsConfigPrefix:  "database_monitoring.metrics.",
 		hostnameEndpointPrefix: "dbm-metrics-intake.",
+		intakeTrackType:        "dbmetrics",
```

Comment:
Hey @justiniso, I know the dbmetrics has not officially onboarded onto event-platform, but looking forwards, would you confirm `dbmetrics` is the track name you will use?

Reply:
Since this is not actually defined anywhere other than feature branches, I think we should switch this to `dbmmetrics` to remain consistent with our intake endpoint.